File: fix.py
import re
with open('js/data.js', 'r', encoding='utf-8') as f:
    text = f.read()

def replacer(match):
    inner = match.group(1).replace('\"', '\'')
    return f'name: \"{inner}\", brand:'

text = re.sub(r'name:\s*\"(.*?)\",\s*brand:', replacer, text)

with open('js/data.js', 'w', encoding='utf-8') as f:
    f.write(text)

File: fix.cs
using System;
using System.IO;

public class Program {
    public static void Main() {
        string[] lines = File.ReadAllLines("js/data.js");
        for(int i=0; i<lines.Length; i++) {
            int idx1 = lines[i].IndexOf("name: \"");
            int idx2 = lines[i].IndexOf("\", brand:");
            if(idx1 >= 0 && idx2 > idx1) {
                int start = idx1 + 7;
                string inner = lines[i].Substring(start, idx2 - start);
                inner = inner.Replace("\"", "'");
                lines[i] = lines[i].Substring(0, start) + inner + lines[i].Substring(idx2);
            }
        }
        File.WriteAllLines("js/data.js", lines, System.Text.Encoding.UTF8);
    }
}
